docs/source/alfacase/02_alfacase_syntax.rst

```diff
@@ -0,0 +1,216 @@
+.. _alfacase-syntax:
+
+ALFAcase Syntax
+===============
+
+
+An :program:`ALFAcase` file is a text-based file that must be written following the YAML syntax, only a restricted subset of the YAML specification
+is followed and the following section and this section covers the support YAML syntax how to configure a project.
```

Comment:
```suggestion
is supported and the this section and the next cover the differences.
```

But I'm not sure we should add these here: while I appreciate the intention, I think a link to the Strict YAML docs is more effective and can replace this entire section.

Reply:
I don't know if a link to the StrictYAML would be good for our users since the StrictYAML docs are focused on the DEV using the lib, with mentions and instructions on how to extract and manipulate the data, many details that out of the scope of our user that only needs to fill a file.

I also don't wanna link a cheatsheet with YAML syntax since only a subset of the specification is followed by strictyaml,  and in my view, our users just need to know how to write a "string", "number", "boolean" and a "list with numbers".

Reply:
> I don't know if a link to the StrictYAML would be good for our users since the StrictYAML docs are focused on the DEV using the lib

Ahh good point, I misremembered the docs, I thought they had a quick YAML primer but they don't.

> I also don't wanna link a cheatsheet with YAML syntax since only a subset of the specification is followed by strictyaml, and in my view, our users just need to know how to write a "string", "number", "boolean" and a "list with numbers".

I understand, but I also don't want to replicate YAML's documentation here. But OK, I also couldn't find any primer/tutorial which was generic enough, so let's keep it. 👍